package.json

```diff
@@ -12,11 +12,15 @@
 	"engines": {
 		"node": ">=6"
 	},
+	"os": [
+		"win32"
+	],
```

Comment:
I prefer a runtime check, as this will prevent a package to support multiple systems where this package is support for one of those.